### PR TITLE
Update repo names for sclorg examples

### LINKED ISSUE #19998: Update example repos that were moved to sclorg
Many of the GitHub repos for our example templates (ex: `django-ex`) have been moved to sclorg.

These should be updated so we don't rely on the redirect.

### PR DESCRIPTION
**Pull request by adambkaplan**
Fix github repo organization for example code used for quickstarts and tests:
- nodejs
- django
- cakephp
- ruby
- rails
- dancer

Note that imagestreams and quickstart templates are excluded.

Fixes #19998

### CODE REVIEW

**Comment by adambkaplan** [comment]
/assign @bparees 

Per our conversation imagestreams have not been updated. I also have not changed the quickstart templates because these need to be updated in openshift/library first. I believe we have a bot that syncs the templates.

File: pkg/oc/cli/cli.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ var (
     To create a new application, login to your server and then run new-app:
 
         %[1]s login https://mycluster.mycompany.com
-        %[1]s new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git
+        %[1]s new-app centos/ruby-22-centos7~https://github.com/sclorg/ruby-ex.git
```
Comment (bparees) [inline pkg/oc/cli/cli.go]: if we're touching this, let's bring it up to a newer image version too.

File: pkg/oc/cli/newapp/newapp.go [hunk 2]
```diff
@@ -96,7 +96,7 @@ var (
 	  %[1]s %[2]s . --docker-image=repo/langimage
 
 	  # Create a Ruby application based on the provided [image]~[source code] combination
-	  %[1]s %[2]s centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git
+	  %[1]s %[2]s centos/ruby-22-centos7~https://github.com/sclorg/ruby-ex.git
```
Comment (bparees) [inline pkg/oc/cli/newapp/newapp.go]: if we're touching this, let's bring it up to a newer image version too.

File: pkg/oc/cli/newbuild/newbuild.go [hunk 3]
```diff
@@ -42,7 +42,7 @@ var (
 	  %[1]s %[2]s . --docker-image=repo/langimage
 
 	  # Create a NodeJS build config based on the provided [image]~[source code] combination
-	  %[1]s %[2]s openshift/nodejs-010-centos7~https://github.com/openshift/nodejs-ex.git
+	  %[1]s %[2]s openshift/nodejs-010-centos7~https://github.com/sclorg/nodejs-ex.git
```
Comment (bparees) [inline pkg/oc/cli/newbuild/newbuild.go]: if we're touching this, let's bring it up to a newer image version too.

File: pkg/oc/cli/newbuild/newbuild.go [hunk 4]
```diff
@@ -70,13 +70,13 @@ var (
 To build from an existing image stream tag or Docker image, provide the name of the image and
 the source code location:
 
-  %[1]s %[2]s openshift/nodejs-010-centos7~https://github.com/openshift/nodejs-ex.git
+  %[1]s %[2]s openshift/nodejs-010-centos7~https://github.com/sclorg/nodejs-ex.git
```
Comment (bparees) [inline pkg/oc/cli/newbuild/newbuild.go]: if we're touching this, let's bring it up to a newer image version too.

File: pkg/oc/cli/requestproject/request_project.go [hunk 5]
```diff
@@ -61,7 +61,7 @@ const (
 	requestProjectNewAppOutput = `
 You can add applications to this project with the 'new-app' command. For example, try:
 
-    %[1]s new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git
+    %[1]s new-app centos/ruby-22-centos7~https://github.com/sclorg/ruby-ex.git
```
Comment (bparees) [inline pkg/oc/cli/requestproject/request_project.go]: if we're touching this, let's bring it up to a newer image version too.

**Pull request merged**